src/assets/locales/resources_en_US.json

```diff
@@ -0,0 +1,10 @@
+{
+  "sky_image_do_test": {
```

Comment:
Should this be `"sky_image_do_text"`?

Reply:
Also, I know we're inconsistent, but the intended prefix for the strings is `skyux_`.
See: https://github.com/blackbaud/skyux-list-builder/blob/master/src/assets/locales/resources_en_US.json#L2